ansible/inventory/group_vars/apps.yaml

```diff
@@ -42,6 +42,8 @@ security_groups:
   publish_dispatcher_elb:
     tag_name: AEM Publish Dispatcher ELB Security Group
     inbound_cidr_ip: overwrite-me
+    extra_groups:
+      - overwrite-me
```

Comment:
Since it's a secgroup, it's better to use the sg-1234567 example, otherwise user will think this is a freeform string.

Reply:
Done.